### ISSUE TITLE
[Feature Request] Create shortcuts to specific instances.

### ISSUE THREAD

**Issue by GenuineSounds** [MultiMC/MultiMC5#1585]
Create the ability to create shortcuts to auto-launch instances/profiles. Something I know would be really nice for people that would also like to add their instances to Steam to launch directly instead of via the Launcher GUI.

**Comment by wlhlm** [comment]
Duplicateof #253. Also tracked in [workflowy](https://workflowy.com/s/2EyDMcp7CU).

@peterix Maybe add a link to workflowy somewhere (I couldn't find one other than googeling). Many issues just say "Moved to workflowy" and you don't know their status (implemented, planned for next release, dropped, to do, ...).

**Comment by GenuineSounds** [comment]
@wlhlm Yeah wow, I tried googling a LOT to the same affect but with the official launcher. Turns out this feature did exist in MultiMC at one point but was not included in the rewrite.

Apparently there is a command-line argument for passing in an instance id, but I can't get any of the command-line arguments to work in the latest stable build of MultiMC (v5 0.4.11-751). Is there a know problem with the arguments parser or perhaps it's Windows (10)?

![Screenshot](https://cloud.githubusercontent.com/assets/945942/15894065/a39629cc-2d38-11e6-8438-09dd902287d4.png)
